cms/templates/import.html

```diff
@@ -115,7 +115,7 @@ <h3 class="title">${_("Uploading")}</h3>
 
             <li class="item-progresspoint item-progresspoint-unpack is-started">
               <span class="deco status-visual">
-                <i class="icon fa fa-cog"></i>
+                <i class="icon fa fa-cog fa-spin"></i>
```

Comment:
Hmm, I'm actually not sure - I can't remember if I added that in, but most likely it was an accident while performing one of the hard rebases. The import form in master seems to look and behave fine, so I'll get rid of it.